### DISCUSSION TITLE
OpenVPN for Xamarin Forms/.NET MAUI

### DISCUSSION THREAD

**Discussion by VegieIsCute** [schwabe/ics-openvpn#1448] [General]
Hello, I'm trying to make an Android VPN using Microsoft's .NET MAUI (https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui).

I can't seem to find much information on this topic. I want to know how I would do this.

**Comment by schwabe** [comment]
I have next no experience with Xamarin, so I cannot answer you that. You can control this app via AIDL. That might be workable?

**Reply by VegieIsCute** [reply]
How would I do this, like Java examples I can port them to .NET and see

**Reply by schwabe** [reply]
the remoteExample is already in this repo
